packages/platform-feeds/index.js

```diff
@@ -257,10 +246,10 @@ function buildFeedObject(dateNum, item) {
     date: item.date,
     datenum: dateNum,
     tags: item.categories,
-    text: item.summary,
-    html: item.summary,
-    brief_text: item.description,
-    brief_html: item.description,
+    brief_text: item.summary,
+    brief_html: item.summary,
+    text: item.description,
+    html: item.description,
```

Comment:
Same as above: why have a `text` key here that duplicates the (HTML) data from the other key?